packages/mollie-integration/src/MollieAPMCustomForm.test.tsx

```diff
@@ -59,44 +57,27 @@ describe('MollieAPMCustomForm', () => {
     });
 
     it('should empty render', () => {
-        const container = mount(
+        const { container } = render(
             <MollieAPMCustomFormTest method={{ ...method, initializationData: {} }} />,
         );
 
-        expect(container.isEmptyRender()).toBe(true);
+        expect(container).toBeEmptyDOMElement();
     });
 
     it('should render MollieAPMCustomForm', () => {
-        const container = mount(<MollieAPMCustomFormTest method={method} />);
+        const { container } = render(<MollieAPMCustomFormTest method={method} />);
+        const dropdownTrigger = container.querySelector('.dropdownTrigger');
```

Comment:
would be better to use RTL-style queries, if it's possible. It's not a mandatory change request.